packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -1671,6 +1674,130 @@ class _AnimatedRotationState extends ImplicitlyAnimatedWidgetState<AnimatedRotat
   }
 }
 
+/// Animated version of [Transform.translate] which automatically transitions the child's
+/// offset relative to its normal position whenever the given offset changes.
+///
+/// The translation is expressed as an [Offset] scaled to the child's size. For
+/// example, an [Offset] with a `dx` of 0.25 will result in a horizontal
+/// translation of one quarter the width of the child.
+///
+/// {@tool snippet}
+///
+/// This code defines a widget that uses [AnimatedOffset] to translate a [FlutterLogo]
+/// up or down by the amount of it's height with each press of the corresponding button.
+///
+/// ```dart
+/// class LogoTranslate extends StatefulWidget {
+///   const LogoTranslate({Key? key}) : super(key: key);
+///
+///   @override
+///   State<LogoTranslate> createState() => LogoTranslateState();
```

Comment:
Should all of these names be LogoOffset* instead of LogoTranslate* to match the widget?